# request constants
uint8 VELOCITY_SCALE=0
uint8 VELOCITY_OBSTACLE=1
uint8 EXTERNAL=2
uint8 BEHIND_ROBOT=3
uint8 PREDICTED_GOAL=4
# request fields
uint8                               type
float64[]                           predict_times
int64[]                             ids
---
# response fields
human_path_prediction/PredictedPoses[]    predicted_humans_poses
